samples/Calling/src/app/views/CallScreen.tsx

```diff
@@ -37,6 +37,12 @@ export const CallScreen = (props: CallScreenProps): JSX.Element => {
       });
       adapter.on('error', (e) => {
         console.error(e);
+        // TODO: Handle other error types from adapter event emitter. Error type does not
+        // have properties 'operation' and 'error'
+        // Do not call error handler when start screen sharing fails
+        if (e['operation'] === 'Call.startScreenSharing' && e['error']['_code'] === 400) {
+          return;
+        }
         onCallError(e);
```

Comment:
Great find.

This PR will handle the bug you're fixing, but I think we might want to go further (OK in a follow up PR) -- currently, *any* error from the adapter will kick us to the error screen. I think this should be an allowlist instead -- log and squelch errors here, except for a few really bad ones (I can't think of any bad enough ones at present and we haven't seen this behaviour in all our self hosting, so I expect that there aren't actually many errors where we'd want to leave the call).

Reply:
Sounds good. I am just squelching based on error target being 'Call.startScreenSharing' as I can't really use the string stack intelligently.